examples/next-prisma-todomvc/src/pages/[filter].tsx

```diff
@@ -278,8 +278,7 @@ export default function TodosPage({
               <Link
                 href="/all"
                 className={clsx(
-                  !['active', 'completed'].includes(filter as string) &&
```

Comment:
https://typescript-eslint.io/rules/no-unnecessary-type-assertion

`filter` is already a `string`. The `as` type assertion does nothing.